web/src/containers/activity/StandardsAndConditions.js

```diff
@@ -1,247 +1,75 @@
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.modularity.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.modularity}
-          onSync={getHandler(setModularity)}
-          editorClassName="rte-textarea-l"
-        />
-      </div>
-
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.mita.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.mita}
-          onSync={getHandler(setMITA)}
-          editorClassName="rte-textarea-l"
-        />
-      </div>
-
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.industryStandards.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.industryStandards}
-          onSync={getHandler(setIndustryStandards)}
-          editorClassName="rte-textarea-l"
-        />
-      </div>
-
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.leverage.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.leverage}
-          onSync={getHandler(setLeverage)}
-          editorClassName="rte-textarea-l"
-        />
-      </div>
-
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.businessResults.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.businessResults}
-          onSync={getHandler(setBusinessResults)}
-          editorClassName="rte-textarea-l"
-        />
-      </div>
-
-      <div className="ds-u-margin-bottom--6">
-        <Instruction
-          source="activities.standardsAndConditions.reporting.instruction"
-          headingDisplay={{
-            level: 'h6',
-            className: 'ds-h5'
-          }}
-        />
-        <RichText
-          content={activity.standardsAndConditions.reporting}
-          onSync={getHandler(setReporting)}
-          editorClassName="rte-textarea-l"
+  setDoesNotSupport,
+  setSupport
+}) => (
+  <Fragment>
+    <h4 className="ds-h4">Standards & Conditions</h4>
+    <div className="ds-u-margin-bottom--6 ds-u-margin-top--3">
+      <h5 className="ds-h5 ds-u-margin-bottom--3">
```

Comment:
These probably shouldn't be headings, because they're really describing the text input instead of introducing a section. We can make them `p` tags with `ds-h5` styling, or we could also make them `label`s for their text editors. (I don't think either of the textareas here have labels.) The latter would be more accessible (and they should retain the styling).